src/components/SchedulingForm.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect }  from 'react'
+import styled from 'styled-components';
+ 
+export const SchedulingForm = (props) => {
+	const [amount, setAmount] = useState('');
+	const [speed, setSpeed] = useState('');
+	const [type, setType] = useState('');
+	const [validInput, setValidInput] = useState(false);
+
+	useEffect(() => {
+		setType(props.products[0].name)
+	}, [])
+
+
+	const addScheduledBatch = (e) => {
+        e.preventDefault(); 
+
+		if (validInput) {
+			let data = {
+				"speed": speed,
+				"type": type.toUpperCase().replace(" ", "_"),
+				"amount": amount
+			};
+
+			fetch("http://localhost:8080/api/scheduled-batches", {
+				method: 'POST',
+				headers: {'Content-Type': 'application/json'},
+				body: JSON.stringify(data)
+			}).then(response => {
+				if(response.status === 200){
+					setAmount("");
+					setSpeed("");
+					props.update();
+				}
+			})
+		}
+    }
+
+    const onSpeedChanged = (e) =>{
+		setSpeed(e.target.value);
+
+
+		//Check that speed only contains numbers
+		const regex = /^\d+$/;;
+		setValidInput(regex.test(e.target.value));
+
+		//Check that value is within acceptable range
+		if (e.target.value > 0 && e.target.value <= getMaxSpeed(type)) {
+			setValidInput(true);
+		} else {
+			setValidInput(false);
+		}
+
+    }
+
+    const onAmountChanged = (e) =>{
+		setAmount(e.target.value);
+    }
+
+    const changeBeerType = (e) => {
+		setType(e.target.value);
+    }
+
+	const getMaxSpeed = (selectedProduct) => {
+		let result = "?";
+		props.products.forEach(product => {
+			if (selectedProduct === product.name) {
+				result = product.speed;
+			}
+
+		});
+
+		return result;
+	}
+
+	return (
+		<Styleddiv>
+			<Styledform>
+				<select onChange={changeBeerType}>
+					{props.products.map((product) => (
+						<option value={product.name} key={product.name}>
+							{product.name} 
+						</option>
+					))}
+				</select>
+
+				<input placeholder="Amount..." value={amount} onChange={onAmountChanged}/>
```

Comment:
```suggestion
				<input placeholder="Amount" value={amount} onChange={onAmountChanged}/>
```
Det stikker lidt ud at det er den eneste med "..."